### PR TITLE
Add other continuous delivery options

### PR DESCRIPTION
**Pull request by MaikuB**
This is my attempt to makes changes as raised in #2699. Tried to update the name of the page, firebase.json etc to remove mention of fastlane to keep it clean/consistent. Note that I've kept the link in the testing page to point to fastlane guide. A staging site has been deployed here for convenience https://maikub-flutter-website.firebaseapp.com

### CODE REVIEW

File: src/docs/deployment/cd.md [hunk 1]
```diff
@@ -1,48 +1,50 @@
 ---
-title: Continuous Delivery using fastlane with Flutter
-description: How to use fastlane to automate continuous building and releasing of your Flutter app.
+title: Continuous Delivery with Flutter
```
Comment (sfshaza2) [inline src/docs/deployment/cd.md]: Delivery => delivery
(We use sentence case for titles, though I see we missed this title before...)

File: src/docs/deployment/cd.md [hunk 2]
```diff
@@ -166,3 +168,15 @@ of fastlane in action. Also see the Flutter framework repository's
 [Cirrus script]({{site.github}}/flutter/flutter/blob/master/.cirrus.yml).
 
 [fastlane CI documentation]: https://docs.fastlane.tools/best-practices/continuous-integration
+
+## Other services
+
+The following are some other options available to help automate the delivery of your application
```
Comment (sfshaza2) [inline src/docs/deployment/cd.md]: Add a period to the end of the sentence.

File: src/docs/testing/index.md [hunk 3]
```diff
@@ -85,7 +85,7 @@ For information on running tests on various continuous integration services,
 see the following: 
 
 * [Continuous Delivery using fastlane with
```
Comment (sfshaza2) [inline src/docs/testing/index.md]: Continuous Delivery => Continuous delivery

**Comment by MaikuB** [comment]
Finished the tweaks now

**Pull request merged**